proxy.go

```diff
@@ -35,6 +35,7 @@ type ProxyHttpServer struct {
 	harLog            *har.Har
 	harLogEntryCh     chan harReqAndResp
 	harFlushRequest   chan string
+	HarFlushToMemory  chan chan *har.Har
```

Comment:
I am using the HAR format because I don't this it is necessary to turn this information in another format. See my comment at https://github.com/abourget/goproxy/pull/2#discussion_r36582630 to see how I am using this information.